site/docs/data.md

```diff
@@ -71,6 +71,34 @@ For example, to create a data source named `myData`, use the following data
 }
 ```
 
+Then you can load it with vegaEmbed modying the [Vega View](https://vega.github.io/vega/docs/api/view/#data) using the insert or change methods.
+
+```js 
+const embed_opt = {"mode": "vega-lite"};
```

Comment:
This is not necessary when you use `$schema`. Let's omit it here.

Reply:
ok